app/code/Magento/Sitemap/Model/SitemapSendEmail.php

```diff
@@ -0,0 +1,109 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Sitemap\Model;
+
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\Framework\Translate\Inline\StateInterface;
+use Magento\Framework\Mail\Template\TransportBuilder;
+use Magento\Store\Model\ScopeInterface;
+use Magento\Backend\App\Area\FrontNameResolver;
+use Magento\Store\Model\Store;
+
+/**
+ *  Sends emails for the scheduled generation of the sitemap file
+ */
+class SitemapSendEmail
```

Comment:
I'd rename the class to the something like `EmailNotification` or `Mail` and the method to `send`  or `sendErrors` to avoid duplicate wording in full method reference. Currently it's `Magento\Sitemap\Model\SitemapSendEmail::sendErrorEmail`

Reply:
ok